smallcloudai/refact-chat-js
# Fix: Adjusted logic for Login and Create Account procedures (semantic changes)

# Fix: Adjusted logic for Login and Create Account procedures

## Description
PRO tier was triggering login button to open up a payment page, now it only runs authentication. Button for account creation works as usual.
I've changed `onLogin()` function to `onLoginOrCreateAccount()` and implemented checks for isLogin flag and plan existence.

## Type of change

## How to Test

- Step 1: Start refact-lsp
- Step 2: Start refact-chat-js
- Step 3: Choose Refact Cloud as Setup Option
- Step 4: Choose between FREE or PRO tiers.
- Step 5: Try clicking "Log In" button with different plans chosen under it. The selection **should not** impact on button's behavior.
- Step 6: Try clicking "Create Account" button with different plans. The selection **should** impact on button's behavior.
